devops/windows/base-docker-wsl/helper/scripts/d-stack-09-wait-removed.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+set -u
+
+pushd .
+  SCRIPT_DIR=$(cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")" && pwd -P)
+  cd ${SCRIPT_DIR}/../..
+
+  source ./conf.env
+
+  declare -a SERVICES=(
+    async-receiver
+    sync-receiver
+    etl
+    controller
+    em
+    linker
```

Comment:
Wont having these app names causes errors?

Reply:
must be removed ----- java apps not running in wsl